packages/gene/src/ioc/react.ioc.tsx

```diff
@@ -108,9 +108,11 @@ export function useCallbackInjection<T, R = void>(
   }
 }
 
-export function withIoc<T>(getContainer: (props: T) => interfaces.Container) {
```

Comment:
`props: T` without specifying more specific type, causes issues in more strict projects